site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/controller/SiteController.java

```diff
@@ -0,0 +1,40 @@
+package uk.ac.ox.ndph.mts.site_service.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+import uk.ac.ox.ndph.mts.site_service.model.Site;
+import uk.ac.ox.ndph.mts.site_service.service.SiteService;
+
+/**
+ * Controller for site endpoint of site-service
+ */
+@RestController()
```

Comment:
shouldn't this be @RestController, not @RestController()

i.e no brackets

Reply:
ahh thats maybe why we missed the reason the tests were failing?? Did Kedar put "/sites" in @RestController instead of @RequestMapping("/sites") ???

Reply:
Fixed Also using the following
@RequestMapping("/sites")
and
@PostMapping()